gradle/spotbugs-exclude.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+  <Match>
+    <Class name="io.opentelemetry.instrumentation.auto.jdbc.normalizer.SimpleCharStream"/>
+    <Not>
+      <Bug code="DM_DEFAULT_ENCODING"/> <!-- ignore default encoding for auto-genned class -->
+    </Not>
+  </Match>
+
+  <Match>
+    <Class name="io.opentelemetry.auto.util.gc.GCUtils"/>
+    <Not>
+      <Bug code="DM_GC"/> <!-- forced GC only used in testing -->
+    </Not>
+  </Match>
+
+  <Match>
+    <Class name="~muzzle\.TestClasses.*"/>
+    <Not>
+      <Bug code="MS_SHOULD_BE_FINAL"/> <!-- final field warnings can be ignored for this test class -->
+    </Not>
+  </Match>
+
+  <Match>
+    <Class name="~HttpServletResponseTest\$.*"/>
+    <Not>
+      <Bug code="SE_INNER_CLASS"/> <!-- inner class serialization warning can be ignored for testing purposes -->
+    </Not>
+  </Match>
+
+  <Match>
+    <Class name="~HttpServletTest\$.*"/>
+    <Not>
+      <Bug code="SE_INNER_CLASS"/> <!-- inner class serialization warning can be ignored for testing purposes -->
+    </Not>
+  </Match>
+
+  <Match>
+    <Class name="~SpymemcachedTest\$.*"/>
+    <Not>
+      <Bug code="SE_INNER_CLASS"/> <!-- inner class serialization warning can be ignored for testing purposes -->
+    </Not>
+  </Match>
+
+  <Match>
+    <Class name="~TestServlets.*"/>
```

Comment:
We should fix it instead of silencing. The correct way is to pass Callable, as `HttpServerTest#controller` expects. Which now means just lambda.